llvm/test/SYCLLowerIR/group_local_memory_debug.ll

```diff
@@ -0,0 +1,95 @@
+; RUN: opt -S -sycllowerwglocalmemory < %s | FileCheck %s
```

Comment:
Please consider using `-debugify` pass in this test. It will automatically attach debug metadata to every instruction. Here is one example: https://github.com/intel/llvm/blob/sycl/llvm/test/SYCLLowerIR/esimd_lower_debug_info.ll

Unless you're trying to check something that is not handled by `debugify`.

Reply:
SYCLLowerWGLocalMemoryPass creates new global variable as storage for WG static local memory.
This test should check that SYCLLowerWGLocalMemoryPass also generates proper debug info for that new global variable.

Reply:
@DenisBakhvalov I've tried to use -debugify pass, but there is some specific debug info that is required for SYCLLowerWGLocalMemoryPass and is not generated by Debugify pass (`inlinedAt` debug info for __sycl_allocateLocalMemory call instruction, `uint8_t` type info).

Reply:
@mlychkov ok, thanks!